src/status_im/subs/wallet/wallet.cljs

```diff
@@ -88,6 +88,11 @@
  :<- [:wallet/wallet-send]
  :-> :suggested-routes)
 
+(rf/reg-sub
+ :wallet/wallet-bridge-to-chain-id
```

Comment:
I was thinking this sub was pretty straightforward, so it might not need a test (or at least it is not that critical). Let me know if you think we should add it, and I can work on it 💪

Reply:
I think we just add them for all subs so that if they need to be extended/modified we can do so safely 👌